ox-hugo.el

```diff
@@ -506,7 +518,37 @@ are \"toml\" and \"yaml\"."
                                        value)
                                       (t
                                        (org-hugo--wrap-string-in-quotes value)))))))))
-    (concat sep front-matter sep)))
+    (concat sep front-matter (if (alist-get "menu" data) (org-hugo--collect-menu-metadata info format) (org-hugo--collect-menu-metadata info format) ) sep)))
+
+(defun org-hugo--collect-menu-metadata (info format)
+  "collect all the menu-related metadata and return a nested plist of the values
+to be used by toml and yaml frontmatter creators."
+  ;;(message "info: %s" info)
+  (let* ((menu-atts '(:hugo-menu-parent "parent" :hugo-menu-weight "weight"))
+         (menu-name (plist-get info :hugo-menu-name))
+         (menu-identifier (plist-get info :title))
+         (menu-weight (plist-get info :hugo-menu-weight))
+         (menu-parent (plist-get info :hugo-menu-parent))
+         ;; (atts-plist (cl-loop for att in (plist-get-keys menu-atts)
+         ;;                      collect (plist-get menu-atts att) 
+         ;;                      collect  (plist-get info att)))
+         ;;(menu-plist (plist-put '() menu-name `("parent" ,menu-parent "weight" ,menu-weight)))
+         )
+    
+    
+    (if (string= format "yaml")
+        (let ((indent "    "))
+          (format "\nmenu:\n  %s:\n    %s: %s\n    %s: %s\n    %s: %s\n"
```

Comment:
Looks like you plan to have just one "menu name" per post? I will try to get this working for just that for now. Supporting multiple menu names (and their hierarchies) dynamically seems a bit complicated.